client/my-sites/site-settings/main.jsx

```diff
@@ -67,7 +56,6 @@ export class SiteSettingsComponent extends Component {
 	}
```

Comment:
Just noticed we're not even using the `getStrings()`. Care to clean it up? It would also rid us of importing`i18n`.